source/ui/ActivityItem/index.jsx

```diff
@@ -124,17 +135,23 @@ ActivityItem.defaultProps = {
   status: null,
   plug: null,
   swapData: null,
+  icon: null,
+  name: null,
+  type: ACTIVITY_TYPES.PLUG,
 };
 
 ActivityItem.propTypes = {
-  type: PropTypes.number.isRequired,
+  type: PropTypes.number,
   currency: PropTypes.shape(currencyPropTypes).isRequired,
   to: PropTypes.string,
   from: PropTypes.string,
   amount: PropTypes.number,
   value: PropTypes.number,
   status: PropTypes.number,
-  date: PropTypes.instanceOf(Date).isRequired,
+  date: PropTypes.oneOfType([
```

Comment:
Added this since the connection date is passed as a string for serialization reasons.